spock-core/src/main/groovy/spock/util/EmbeddedSpecCompiler.groovy

```diff
@@ -75,35 +77,77 @@ class EmbeddedSpecCompiler {
   }
 
   List<Class> compileWithImports(@Language('Groovy') String source) {
-    addPackageImport(Specification.package )
+    addPackageImport(Specification.package)
     // one-liner keeps line numbers intact
     doCompile "package apackage; $imports ${source.trim()}"
   }
 
   Class compileSpecBody(@Language(value = 'Groovy', prefix = 'class ASpec extends spock.lang.Specification { ', suffix = '\n }')
-                        String source) {
+                          String source) {
```

Comment:
Are these indentation changes intentional?

Reply:
Result of auto format file, so semi intentional ;)

Reply:
Really?
I thought about it, but couldn't come up with the settings that would produce this.
I know indent length, continuation indent length and aligned with the other parameters.
Which settings combination produces it like that?

Reply:
https://github.com/leonard84/spock/blob/formatter-issue-555/config/idea/Spock.xml

Reply:
Ah, of course, this is aligend + continuation indent.
The `@Language` annoation is for that parameter, so the formatting makes sense of course.